sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/AnalyzeSentiment.java

```diff
@@ -19,9 +19,9 @@ public class AnalyzeSentiment {
     public static void main(String[] args) {
         // Instantiate a client that will be used to call the service.
         TextAnalyticsClient client = new TextAnalyticsClientBuilder()
-            .credential(new AzureKeyCredential("{key}"))
-            .endpoint("{endpoint}")
-            .buildClient();
+                                         .credential(new AzureKeyCredential("{key}"))
```

Comment:
revert?

Reply:
In my IDE, it automatically set the indentation like it. I would keep it.